lib/worker/merge.js

```diff
@@ -20,25 +20,24 @@ api.merge = async ({
   worker, priorityPeers = [], witnesses = worker.consensusState.witnesses,
   basisBlockHeight, peerWitnessParentThreshold
 }) => {
-  if(witnesses.length === 0) {
+  if(witnesses.size === 0) {
     throw new TypeError('"witnesses" must be a non-empty array.');
```

Comment:
```suggestion
    throw new TypeError('"witnesses" must be a non-empty set.');
```

Reply:
I didn't look that closely to see if other TypeErrors need to be updated, please take a quick look.